src/main/resources/application.yml

```diff
@@ -51,6 +51,7 @@ notifications:
   lr-refresh-job-enabled: ${LR_REFRESH_JOB_ENABLED:true}
   completed-job-enabled: ${COMPLETED_JOB_ENABLED:true}
   incompleted-job-enabled: ${INCOMPLETED_JOB_ENABLED:true}
+  lr-refresh-job-statement-buffer: ${LR_REFRESH_JOB_STATEMENT_BUFFER:200}
```

Comment:
Can we make an estimate of a safe number that'll make better use of the 40MB query limit? e.g. if the average statement is about 4k we could get 10,000 statements in a single query, so could we set this limit to 1000 to make the buffer iteration as efficient as possible?